Auth (#5336)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/mfa.md

---
title: "Multi-factor authentication"
description: "Configure dbt Cloud MFA"
id: "mfa"
sidebar: null
---

dbt Cloud provides multiple options for multi-factor authentication (MFA). MFA provides an additional layer of security on top of username and password. The available MFA methods are:
- SMS verification code
- Authenticator app
- Webauthn-compliant security key

## Configuration

> You may have one of the three MFA methods configured per user. These are enabled at the user level, not the account level.

Author: ```suggestion
You can only have one of the three MFA methods configured per user. These are enabled at the user level, not the account level.
```

1. Navigate to the **Account settings** and under **Your profile** click on **Password & Security**. Click **Enroll** next to the preferred method.

<Lightbox src="/img/docs/dbt-cloud/mfa-enrollment.png" title="Select MFA enrollment method" />

> Choose the next steps based on your enrollment selection:

Author: ```suggestion
Choose the next steps based on your preferred enrollment selection:
```

<expandable alt_header="SMS verification code" >

2. Select your country code, enter your phone number in the field, and click **Continue**.

<Lightbox src="/img/docs/dbt-cloud/sms-enter-phone.png" title="Enter your phone number information" />

3. You will receive an SMS message with a six digit code. Enter the code in dbt Cloud.

<Lightbox src="/img/docs/dbt-cloud/enter-code.png" title="Enter the 6-digit code" />

</expandable>

<expandable alt_header="Authenticator app" >

> 2. Open your preffered authentication app (like Google Authenticator) and scan the QR code.

Reviewer: nit: preferred

Author: Good catch!
```suggestion
2. Open your preferred authentication app (like Google Authenticator) and scan the QR code.
```
